src/main/java/com/wfector/command/CommandRunner.java

```diff
@@ -25,39 +25,39 @@ public CommandRunner(ChestShopNotifier plugin) {
     public boolean onCommand(final CommandSender sender, Command cmd, String label, String[] args) {
         if(args.length == 0) {
             Help.SendDialog(sender);
```

Comment:
You are still calling a method here statically that isn't static. You need to do something like `new Help(this).sendDialog(sender)`. Your IDE should warn you about this. ;)

Reply:
It did not, it gave me an error when using `new Help(this).sendDialog(sender)` 
"Help (com.wfector.notifier.ChestShopNotifier) in Help cannot be applied to (com.wfector.command.CommandRunner)"
 

Reply:
`new Help(plugin).SendDialog(sender);` works though